### PR TITLE
#170670073 User can be able to make a round trip request

### PR DESCRIPTION
**Pull request by nziokaivy**
#### What does this PR do?

Allows a user to make a round trip request.
As a requester, I should be able to select 'Round-trip'.Then provide my start date and target return date as well as the origin and destination.

#### Description of Task to be completed?

 - [x] Set up round trip redux architecture
 - [x] Add create trip request functionality
 - [x] Add validations
 - [x] Add tests
 - [ ] Add styling

#### How should this be manually tested?

- Clone this respository using the command `git clone https://github.com/andela/team-odd-bn-frontend.git`
- Then navigate into the project `cd team-odd-bn-frontend`
- Switch to the round trip branch using the command: git checkout `ft-round-trip-170670073`
- Run the command `npm i` to install all the packages
- Run npm run dev to run the server
- Finally, navigate to the round trip page by navigating to the route `/trips/roundtrip`

#### Any background context you want to provide?

As a user, I should be able to choose any trip type. As a user, I should be able to create a one way, round trip or multi-city trip request

#### What are the relevant pivotal tracker stories?
[#170670073](https://www.pivotaltracker.com/story/show/170670073)

### CODE REVIEW

File: src/containers/trips/RoundTrip.js [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+      reason,
+      startDate,
+      returnDate,
+      formErrors,
+    } = this.state;
+
+    console.log('props', this.props);
+
+    return (
+    // <DashBoard>
```
Comment (hezronkimutai) [inline src/containers/trips/RoundTrip.js]: Hi @ivy, remove this comment

File: src/containers/trips/RoundTrip.js [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                    onChange={this.handleChangeInput}
+                    value={originId}
+                    name="originId"
+                  >
+                    <option>Origin</option>
+                    { cities && cities.data.map((item, index) => (
+                      <option
+                        key={index}
+                        value={item.id}
+                        // selected={(city && item.city === city.data.originId)}
```
Comment (hezronkimutai) [inline src/containers/trips/RoundTrip.js]: This comment too.

**Review by hezronkimutai — COMMENTED** [review]
In your UI implementation, remove the underline in the left part of the form.



Remove underline in the <LINK>. generally, make it match the mockup.

Also, don't forget  to work on the responsiveness.

Screen Shot 2020-01-17 at 08 07 19

**Review by hezronkimutai — APPROVED** [review]
Nice work @nziokaivy

**Review by shemaeric — APPROVED** [review]
Good work @nziokaivy

**Pull request merged**